docs/reference.md

```diff
@@ -95,6 +101,29 @@ response.json = {'some': 'dict'}
 response.json = [{'some': 'dict'}, {'another': 'one'}]
 ```
 
+### Multipart
+
+Responsible of the parsing of `request.body`.
+
+#### Methods
+
+- **parse(content_type: str)**: returns a tuple
```

Comment:
Not sure the name is that good: it should imho reflects the two steps workflow, i.e. "init" then "feed_data" as many times as needed. So maybe `init`.
Or we use the `__init__` (which does not return anything then), and we retrieve form and files `self._form = parser.form`.